packages/sanity/src/desk/panes/document/DocumentPaneContext.ts

```diff
@@ -66,7 +64,7 @@ export interface DocumentPaneContextValue {
   setTimelineMode: (mode: TimelineMode) => void
   setTimelineRange(since: string | null, rev: string | null): void
   source?: string
-  timeline: Timeline
+  timelineController$: Subject<TimelineController>
```

Comment:
Does the outside world need this to be a Subject? (E.g. is there anything externally that calls `.next()` on it, or could it be an `Observable<TimelineController>` instead?

Reply:
It doesn't – that's been updated!